src/components/report-filter-controls.tsx

```diff
@@ -154,28 +154,32 @@ function ReportFilterControls({ controls }: ReportFilterControlsProps): VNode {
                   <div className="usa-radio">
                     <input
                       type="radio"
-                      id="funnel-mode-overall"
+                      id="funnel-mode-blanket"
                       name="funnelMode"
-                      value={FunnelMode.OVERALL}
-                      checked={funnelMode === FunnelMode.OVERALL}
+                      value={FunnelMode.BLANKET}
+                      checked={funnelMode === FunnelMode.BLANKET}
                       className="usa-radio__input"
                     />
-                    <label htmlFor="funnel-mode-overall" className="usa-label usa-radio__label">
-                      Overall
+                    <label htmlFor="funnel-mode-blanket" className="usa-label usa-radio__label">
+                      Blanket
                     </label>
+                    <span className="margin-left-1">The funnel starts at the welcome step</span>
```

Comment:
Might be a good use-case for an `aria-describedby` of `input` (in combination with the label)?

https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html

>a label describes the essence of an object, while a description provides more information that the user might need.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

Reply:
sure, added in 8e0b91ea72021b6d29731e0f9c162b04c863e6a8